platform-tests/src/test/java/org/junit/platform/reporting/legacy/xml/LegacyXmlReportGeneratingListenerTests.java

```diff
@@ -317,10 +316,11 @@ void writesHostNameAndTimestamp(@TempDir Path tempDirectory) throws Exception {
 		engine.addTest("test", () -> {
 		});
 
-		LocalDateTime now = LocalDateTime.parse("2016-01-28T14:02:59.123");
-		ZoneId zone = ZoneId.systemDefault();
+		String timestamp = "2016-01-28T14:02:59Z[Europe/London]";
```

Comment:
❓ That's the format you want in the XML? Could you please double check that tools that read this XML report, e.g. Jenkins, don't choke on this?

Reply:
Yep going to do the same for pytest too https://github.com/pytest-dev/pytest/issues/7662#issuecomment-677679097